### ISSUE TITLE
NeuButton the ability to take Children and onClick

### ISSUE THREAD

**Issue by GeorgeWL** [dev-saeed/neumorphic-ui#1]
Just fits better into the pattern I'm more used to seeing

**Comment by dev-saeed** [comment]
Yes. I have added new docs for NeuButton component and it now includes`onClick` as a prop as well. For now, I haven't added children to NeuButton but I will soon add this too so that users can add icons too.

**Comment by GeorgeWL** [comment]
> so that users can add icons too.

Yeah that entirely my reason for it.

**Comment by dev-saeed** [comment]
Thanks for your suggestion. I will be soon adding it too.
